### PR TITLE
Cover ShowTBTRequest with UT

### PR DESCRIPTION
**Pull request by indlu**
Cover ShowTBTRequest by unit tests

### CODE REVIEW

**Comment by indlu** [comment]
@istoilovgithub , @okozlovlux, @dcherniev please, review.

File: src/components/application_manager/test/commands/mobile/show_constant_tbt_request_test.cc [hunk 1]
```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright (c) 2016, Ford Motor Company
```
Comment (okozlovlux) [inline src/components/application_manager/test/commands/mobile/show_constant_tbt_request_test.cc]: @indlu , 2017

Comment (indlu) [inline src/components/application_manager/test/commands/mobile/show_constant_tbt_request_test.cc]: @okozlovlux, fixed in cc70fc9

**Comment by indlu** [comment]
@istoilovgithub , @dcherniev , gentle reminder

**Pull request merged**